lib/comment.rb

```diff
@@ -0,0 +1,30 @@
+require 'pg'
+require_relative 'database_connection'
+
+class Comment 
+
+  attr_reader :id, :comment, :tweet_id, :user_id
+
+  def initialize(id:, comment:, tweet_id:, user_id:)
+    @id = id 
+    @comment = comment 
+    @tweet_id = tweet_id
+    @user_id = user_id
+  end 
+
+  def self.where(tweet_id:)
+    result = DatabaseConnection.query("SELECT * FROM comment WHERE tweet_id = '#{tweet_id}';")
+    result.each { |comment|
+      Comment.new(id: comment['id'], comment: comment['comment'], tweet_id: comment['tweet_id'], 
```

Comment:
When you have more than a few words of code in a block, I think maybe you could put it into a do/end rather than {}